packages/feedback-service/src/types.d.ts

```diff
@@ -0,0 +1,37 @@
+declare module '*.graphql';declare module '*.json';
```

Comment:
Each declaration should be on a separate line.

```suggestion
declare module '*.graphql';
declare module '*.json';
```